helm/fiftyone-teams-app/templates/delegated-operator-executor-deployment.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if gt (int .Values.delegatedOperatorExecutorSettings.replicaCount) 0 }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "delegated-operator-executor.name" . }}
+  namespace: {{ .Values.namespace.name }}
+  labels:
+    {{- include "delegated-operator-executor.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.delegatedOperatorExecutorSettings.replicaCount | default 0 }}
```

Comment:
we've already tested that this is greater than 0... this entire deployment gets skipped if the replicaCount is 0...

Reply:
🤷🏼‍♂️